packages/data/src/world/mm/overworld/woods_of_mystery.yml

```diff
@@ -9,7 +9,7 @@
     MEET_KOUME: "soul_npc(SOUL_NPC_KOUME_KOTAKE)"
```

Comment:
We need to add a Koume event specifically for the first day. There can be situations where you could have Woods of Mystery in an entrance not accessible on the first day (back of gorman track), but the game would see you flagged the event and then later see you are on first day. The Kotake Reward region should use this new event instead. Perhaps KOUME_DAY1.